bridges/otelzap/core.go

```diff
@@ -155,17 +158,21 @@ func (o *Core) Write(ent zapcore.Entry, fields []zapcore.Field) error {
 
 	r.AddAttributes(o.attr...)
 	if len(fields) > 0 {
-		r.AddAttributes(convertField(fields)...)
+		r.AddAttributes(convertField(&o.ctx, fields)...)
 	}
 
-	o.logger.Emit(context.Background(), r)
+	o.logger.Emit(o.ctx, r)
 	return nil
 }
 
-func convertField(fields []zapcore.Field) []log.KeyValue {
+func convertField(ctx *context.Context, fields []zapcore.Field) []log.KeyValue {
```

Comment:
Can we return the context instead?

```suggestion
func convertField(fields []zapcore.Field) (context.Context, []log.KeyValue) {
```